### PR TITLE
PAYOSWXP-47: Unzer invoice purchase: if there is no company name in config, paymentmethod should be hidden

### PR DESCRIPTION
**Pull request by amirinterlutions**
The date of birth should no longer be requested on the payment type page as soon as something has been entered in the Company name field.

### CODE REVIEW

File: src/Resources/snippet/en_GB/messages.en-GB.json [hunk 1]
```diff
@@ -101,7 +101,7 @@
             },
             "consentCheckbox": {
                 "introduction": "With the transmission of the data necessary for the completion of the invoice purchase, an identity check and credit check to payolution GmbH, Am Euro Platz 2, 1120 Wien I agree.",
-                "textWithLink": "I can revoke my <a href=\"%link%\" data-url=\"%link%\" data-modal-class=\"payolution\" data-toggle=\"modal\" title=\"Einwilligung\">consent</a>  at any time with effect for the future."
+                "textWithLink": "I can revoke my <a data-toggle=\"modal\" data-bs-toggle=\"modal\" data-url=\"%link%\" href=\"%link%\" title=\"Einwilligung\">consent</a> at any time with effect for the future."
```
Comment (rommelfreddy) [inline src/Resources/snippet/en_GB/messages.en-GB.json:104]: please also change this in the de_DE file

**Review by rommelfreddy — CHANGES_REQUESTED** [review]

**Comment by rommelfreddy** [comment]
we should wait for https://github.com/PAYONE-GmbH/shopware-6/pull/291. Here are a few changes, which would break, if this PR got merged at first. After 291 got merged, we should do a few little adjustments to fit the changes from 291.

File: src/Resources/snippet/en_GB/messages.en-GB.json (hunk 1 above)
Comment (amirinterlutions) [reply src/Resources/snippet/en_GB/messages.en-GB.json:104]: Done! I have made the necessary changes in the de_DE file as well.

**Review by rommelfreddy — APPROVED** [review]

**Comment by rommelfreddy** [comment]
@janteuber 
please merge https://github.com/PAYONE-GmbH/shopware-6/pull/292 at first.
please do not squash the commits on merging!

**Pull request merged**